src/core/manager.h

```diff
@@ -23,6 +23,9 @@ typedef struct Unit Unit;
 /* Enforce upper limit how many names we allow */
 #define MANAGER_MAX_NAMES 131072 /* 128K */
 
+/* On sigrtmin+18, private command */
+#define MANAGER_SIGNAL_COMMAND_DUMP_JOBS 0x500
```

Comment:
maybe define this as `_COMMON_SIGNAL_COMMAND_PRIVATE_BASE + 0` then

Reply:
also, maybe define as anonymous enum too

Reply:
or maybe define as:

```c
enum {
        MANAGER_SIGNAL_COMMAND_DUMP_JOBS = x0500
},

assert_cc(MANAGER_SIGNAL_COMMAND_DUMP_JOBS >= _COMMON_SIGNAL_COMMAND_PRIVATE_BASE);
assert_cc(MANAGER_SIGNAL_COMMAND_DUMP_JOBS <= _COMMON_SIGNAL_COMMAND_PRIVATE_END);
```

Reply:
```
../src/core/manager.h:32:44: warning: comparison between ‘enum <anonymous>’ and ‘enum <anonymous>’ [-Wenum-compare]
   32 | assert_cc(MANAGER_SIGNAL_COMMAND_DUMP_JOBS >= _COMMON_SIGNAL_COMMAND_PRIVATE_BASE);
```

I'll go for the first suggestion

Reply:
oh, that's just dumb of gcc...

Reply:
common-signal.h is missing `#pragma once`, that is all. trivial fix...

Reply:
eheh, completely missed that - fixed

Reply:
> ../src/core/manager.h:32:44: warning: comparison between ‘enum <anonymous>’ and ‘enum <anonymous>’ [-Wenum-compare]
   32 | assert_cc(MANAGER_SIGNAL_COMMAND_DUMP_JOBS >= _COMMON_SIGNAL_COMMAND_PRIVATE_BASE);

btw, just cast the values to (int) before comparing, and the warning should go away